Automattic/pocket-casts-ios
# [Give Rating] Display avg value and no rating label

Fixes #1881 

This PR shows the avg rating value next to the stars and the _No ratings_ label if the podcast doesn't have any ratings yet.

## Layout

| Before | After |
| -------- | ------- |
| ![Image-1](https://github.com/Automattic/pocket-casts-ios/assets/912252/fe16a93e-8f15-4270-8afd-61006ae57b29) | ![Image-2](https://github.com/Automattic/pocket-casts-ios/assets/912252/30b2448c-9580-425c-8afa-6cbfd437a71b) |
| ![Image-3](https://github.com/Automattic/pocket-casts-ios/assets/912252/75553a44-3550-4883-b4f5-0fff77628e35) | ![Image-4](https://github.com/Automattic/pocket-casts-ios/assets/912252/18b75fae-feae-4ee5-84f5-947249ad19c5) |

## To Test

• CI must be 🟢 
• With the _giveRatings_ FF on, check a podcast with no ratings and confirm the _No ratings_ is present
• With the _giveRatings_ FF on, check a podcast with ratings and confirm the avg value is present next to the stars
• With the _giveRatings_ FF off, check a podcast with no ratings and confirm the _No ratings_ is NOT present
• With the _giveRatings_ FF off, check a podcast with ratings and confirm the avg value is NOT present next to the stars

## Checklist
